install/kubernetes/Makefile.values

```diff
@@ -25,10 +25,10 @@ else
     export HUBBLE_RELAY_REPO:=${CI_REGISTRY}/${CI_ORG}/hubble-relay-ci
 endif
 
-export CERTGEN_REPO:=quay.io/cilium/certgen
+export CERTGEN_REPO:=docker.io/giorio94/certgen
 # renovate: datasource=docker depName=quay.io/cilium/certgen
-export CERTGEN_VERSION:=v0.1.12
-export CERTGEN_DIGEST:=sha256:bbc5e65e9dc65bc6b58967fe536b7f3b54e12332908aeb0a96a36866b4372b4e
+export CERTGEN_VERSION:=v0.2.0
+export CERTGEN_DIGEST:=sha256:31dd2fccfeb3e2cf4eb47691ba197b098c71ed4ed04791854cfe8b3793047da6
```

Comment:
These variables still need to be appropriately updated to use the upstream certgen version once released.

Reply:
As 0.2.0 is breaking change, we probably need to double check if renovate config needs any update to restrict versions in old branches.

Reply:
> I am wondering how rennovate can update this image, do we need to run some make command after ?

Yep, it seems to be configured to run it: https://github.com/cilium/cilium/blob/main/.github/renovate.json5#L18

> As 0.2.0 is breaking change, we probably need to double check if renovate config needs any update to restrict versions in old branches.

Yep, already merged: https://github.com/cilium/cilium/pull/32998

Reply:
yeah, rennovate configuration is there,  but I am not sure if it _ever_ works. 

Anyway, it's not related to the goal of this change.

Reply:
I've tried running renovate locally, and it seems to correctly pick the update:

```json
{
  "depName": "quay.io/cilium/certgen",
  "currentValue": "v0.1.12",
  "currentDigest": "sha256:bbc5e65e9dc65bc6b58967fe536b7f3b54e12332908aeb0a96a36866b4372b4e",
  "datasource": "docker",
  "replaceString": "# renovate: datasource=docker depName=quay.io/cilium/certgen\nexport CERTGEN_VERSION:=v0.1.12\nexport CERTGEN_DIGEST:=sha256:bbc5e65e9dc65bc6b58967fe536b7f3b54e12332908aeb0a96a36866b4372b4e",
  "updates": [
    {
      "bucket": "latest",
      "newVersion": "v0.2.0",
      "newValue": "v0.2.0",
      "newMajor": 0,
      "newMinor": 2,
      "updateType": "minor",
      "newDigest": "sha256:169d93fd8f2f9009db3b9d5ccd37c2b753d0989e1e7cd8fe79f9160c459eef4f",
      "branchName": "renovate/all-dependencies"
    }
  ],
  "packageName": "quay.io/cilium/certgen",
  "versioning": "docker",
  "warnings": [],
  "registryUrl": "https://quay.io",
  "currentVersion": "v0.1.12",
  "isSingleVersion": true,
  "fixedVersion": "v0.1.12"
},
```